worker/uploader.go

```diff
@@ -139,6 +139,8 @@ outer:
 			canceledOverdrive := req.done() && req.overdrive && err != nil
 			if !canceledOverdrive && !isClosedStream(err) {
 				u.trackSectorUpload(err, elapsed)
+			} else {
+				u.logger.Debugw("not tracking sector upload metric", zap.Error(err))
```

Comment:
Added this little log line here. I got a bit nervous about what sorts of estimates we ignore so I wanted to make sure we have a way to tell.
The case that I'd like to avoid is that we never track slow hosts because overdrive workers end up saving the day.